### ISSUE TITLE
How possible would it be to build for Android?

### ISSUE THREAD

**Issue by ceopaludetto** [OpenSwiftUIProject/OpenSwiftUI#99]
I love Swift and its expressiveness, and I'm tired of using cross-platform frameworks that either is a mess to maintain (react native) or uses canvas to build the UI (flutter and compose multiplatform), so I've read about JNI and NDK and also found previous attempts to bridge Android APIs into Swift (https://swiftpackageindex.com/PureSwift/Android). Considering that the Graph implementation here can be ported to different backends (GTK, Windows, ...), would it be possible to port to Android as well?

**Comment by Kyle-Ye** [comment]
Yes. As long as we have a complete cross platform implementation of the OG.

The current Apple's AG is implemented using ObjC and C++ and have heavy dependency on CF and ObjectiveC runtime.

Free feel to contribute [here](https://github.com/OpenSwiftUIProject/OpenGraph) and do not forgot to add your test case.

**Comment by Kyle-Ye** [comment]
Update:
Finally have time to make a first step implementing the actual underlying C++ part. 
https://github.com/OpenSwiftUIProject/OpenAttributeGraph/pull/161

**Comment by Kyle-Ye** [comment]
Let's track the potential future Android support here. 

https://github.com/OpenSwiftUIProject/OpenSwiftUI/issues/593
